app/routes/application.js

```diff
@@ -2,13 +2,9 @@ import Route from '@ember/routing/route';
 import { inject as service } from '@ember/service';
 export default Route.extend({
   ajax: service(),
-  async model () {
-    let response = await this.get('ajax').request('https://node-hnapi.herokuapp.com/news', {
-      method: 'GET',
-      data: {
-        page: 1
-      }
+  model () {
+    return this.get('store').query('story', {
+      page: 1
```

Comment:
The query hash also takes a `filter` property which can any of the following values:

```
front-page (defaults to this if none given)
new
active
show
ask
jobs
```